src/main/java/willy/ui/Greet.java

```diff
@@ -0,0 +1,46 @@
+package willy.ui;
+
+/**
+ * Greetings that the bot can do.
+ */
+public class Greet {
+    String originalGreeting;
+    String greeting;
+    String style = "\t________________________________________________________________\n";
```

Comment:
Perhaps you could add the final modifier and change the naming style to uppercase snake case, since these strings are not meant to be changed?